### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
Description: initial commit

UTC:
1. Verified that Solution Pack get installed successfully.
2. Verified that BES Cyber Systems module get installed successfully.
3. Verified that BES Impact Evaluation widget get installed successfully.
4. Verified that On executing scenario, 3 records get created for BES Cyber System module.
5. Verified that BES Impact Evaluation widget executes successfully on any of the record.
6. Verified that BES Impact Evaluation widget successfully calculate BES Impact
7. Verified that Impact Evaluation and Impact Overdue schedule executes successfully and generate mails as expected.

### CODE REVIEW

File: modules/b_e_s_cyber_systems/detail-layout.json
Comment (cs-rohit-khune) [inline modules/b_e_s_cyber_systems/detail-layout.json:1]: Add New for "Playbooks" as per standard module detailed view playbout

File: playbooks/02 - Use Case - NERC CIP-002/BES Cyber System Impact Report Generation.json [hunk 1]
```diff
@@ -0,0 +1,205 @@
+{
+    "@type": "Workflow",
```
Comment (cs-rohit-khune) [inline playbooks/02 - Use Case - NERC CIP-002/BES Cyber System Impact Report Generation.json:2]: Add "SystemWaitForCompletion" tag to the playbook

File: playbooks/02 - Use Case - NERC CIP-002/BES Cyber System Impact Report Generation.json [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    "triggerLimit": null,
+    "name": "BES Cyber System Impact Report Generation",
+    "aliasName": null,
+    "tag": null,
```
Comment (cs-rohit-khune) [inline playbooks/02 - Use Case - NERC CIP-002/BES Cyber System Impact Report Generation.json:6]: Remove mock output from report generation step

File: playbooks/globalVariables.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+        "id": 2,
+        "name": "Default_Email",
```
Comment (cs-rohit-khune) [inline playbooks/globalVariables.json]: remove this GBL variable

File: modules/b_e_s_cyber_systems/mmd.json
Comment (cs-rohit-khune) [inline modules/b_e_s_cyber_systems/mmd.json:1]: Mark list view columns as "Default Gird Columns"

File: info.json [hunk 4]
```diff
@@ -0,0 +1,98 @@
+{
+    "name": "oT-Compliance-NERCCIP-002",
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.4.0",
```
Comment (cs-rohit-khune) [inline info.json]: fsrMinCompatibility should be 7.4.1

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]
Added review comments. Please look into it

File: modules/b_e_s_cyber_systems/detail-layout.json
Comment (cs-ankit-gupta) [reply modules/b_e_s_cyber_systems/detail-layout.json:1]: Incorporated

File: playbooks/02 - Use Case - NERC CIP-002/BES Cyber System Impact Report Generation.json (hunk 1 above)
Comment (cs-ankit-gupta) [reply playbooks/02 - Use Case - NERC CIP-002/BES Cyber System Impact Report Generation.json:2]: Incorporated

File: playbooks/02 - Use Case - NERC CIP-002/BES Cyber System Impact Report Generation.json (hunk 2 above)
Comment (cs-ankit-gupta) [reply playbooks/02 - Use Case - NERC CIP-002/BES Cyber System Impact Report Generation.json:6]: Incorporated

File: playbooks/globalVariables.json (hunk 3 above)
Comment (cs-ankit-gupta) [reply playbooks/globalVariables.json]: Incorporated

File: modules/b_e_s_cyber_systems/mmd.json
Comment (cs-ankit-gupta) [reply modules/b_e_s_cyber_systems/mmd.json:1]: Incorporated

File: info.json (hunk 4 above)
Comment (cs-ankit-gupta) [reply info.json]: Incorporated

**Review by cs-rohit-khune — APPROVED** [review]
Approved

**Pull request merged**